Add SODIMM DDR4 260Pins Socket

blazra opened the pull request:
Hi,

adding SODIMM DDR4 260 pin socket.

- It seems that only the TE Connectivity is manufacturing these and I don't know if the footprint is standardized or not so I wasn't sure how to name it.
- Footprint actually fits 2 versions of the socket - 4.0 mm high and 5.2 mm high.
8.0 mm, 9.2mm high sockets and the reverse types have different footprint.
- I am not sure about the anchor position - in the datasheet there is a cross in the middle of the connector body - is that the correct position?

http://www.te.com/commerce/DocumentDelivery/DDEController?Action=showdoc&DocId=Customer+Drawing%7F2309407%7F3%7Fpdf%7FEnglish%7FENG_CD_2309407_3.pdf%7F2309407-1
http://www.te.com/commerce/DocumentDelivery/DDEController?Action=srchrtrv&DocNm=2309409&DocType=Customer+Drawing&DocLang=English&DocFormat=pdf&PartCntxt=2309409-1

![sodimm_ddr4_socket](https://user-images.githubusercontent.com/511953/44933807-b46d4a00-ad6a-11e8-9608-4bbcc082fb3d.png)

blazra committed:
Add SODIMM DDR4 Socket

poeschlr commented:
Sorry for the extremely late review. Your pull request seems to have been overlooked.

I would suggest to rotate the full footprint to result in pad 1 being on the top left corner.
There is no silk layer outline, please add one where there are no pads in the way (outside the body dimension -> outside fab layer)
I found a few measurements that are not quite right. Details see dimensioned drawing.
![sodim svg](https://user-images.githubusercontent.com/18327350/55355458-a0d4be00-54c8-11e9-9a32-30bd49799d32.png)

blazra committed:
Fixed Fab layer; Rotated; Added silkscreen outline; Changed center to
connector center

blazra commented:
I have rotated the footprint, added silk outline, and fixed the Fab layer.

However, I am not sure about the anchor. I placed it in the center of the whole connector - it looks like it's correct, according to the datasheet.

![image](https://user-images.githubusercontent.com/511953/55375393-3f364300-550c-11e9-8ebc-c77892db8e66.png)

poeschlr commented:
I think the anchor was ok in the first place. This is because the anchor should be the pick and place origin. Right now the pick and place robot would have no chance to grab the part at the place you use now.

The travis estimation really only works well for symetric parts.

blazra commented:
That is true. But the datasheet references the "center of the whole connector" to the tape. So if I keep the original "center for pickup", the anchor would represent the position where to grab the part, but there would be no reference to the tape.

In other words. The original center is the correct position where to grab the part, but the robot doesn't know where it actually is. (Unless it uses computer vision or someone sets the offset manually, but in that case it doesn't matter, I guess)

![image](https://user-images.githubusercontent.com/511953/55441335-b7534600-55aa-11e9-9b2e-5cb676ccc25f.png)

mojca committed:
Delete non-existing unboxed DSUB HD connectors

There are no unboxed DSUB HD connectors with EdgePinOffset9.40mm

See merge request kicad/libraries/kicad-footprint-generator!1088
See merge request kicad/libraries/kicad-footprints!3583

mojca committed:
Fix edge distance for Power Dynamics, Inc. DSUB with 5.0mm footprint

The correct edge distance is 4.94mm rather than 7.48mm.

Closes: kicad/libraries/kicad-footprints#2627
See merge request kicad/libraries/kicad-footprint-generator!1083
See merge request kicad/libraries/kicad-footprints!3583

mojca committed:
Fix pitch of Horizontal HD DSUB to 1.9mm

HD DSUB connector from PDI with 5.0mm footprint uses 1.9mm rather than 1.98mm pitch on the PCB.

Closes kicad/libraries/kicad-footprints#1396
See merge request kicad/libraries/kicad-footprint-generator!1084
See merge request kicad/libraries/kicad-footprints!3583

mojca committed:
Fix pitch of Horizontal HD DSUB to 2.54mm

HD DSUB connector from PDI uses 2.54mm rather than 1.98mm pitch on the PCB.

Closes kicad/libraries/kicad-footprints#1396
See merge request kicad/libraries/kicad-footprint-generator!1084
See merge request kicad/libraries/kicad-footprints!3583

mojca committed:
Update all remaining DSUB connectors

Fix rounding, update broken datasheet URLs, use the new footprint generator.

Closes kicad/libraries/kicad-footprints#2501
See merge request kicad/libraries/kicad-footprint-generator!1081
See merge request kicad/libraries/kicad-footprint-generator!1087
See merge request kicad/libraries/kicad-footprints!3583

zitronenoeffner committed:
Add Texas Instruments WQFN-16 RTE footprint with smaller EP than the existing WQFN-16 footprints

See merge request kicad/libraries/kicad-footprints!3641

Commit:
Gullwing: Generate SOIC-16W_5.3x10.2mm_P1.27mm

See merge request kicad/libraries/kicad-footprints!3608

samueltardieu committed:
Add PowerSSO-16-1EP_3.9x4.9mm_P0.5mm_EP2.5x3.61mm

See merge request kicad/libraries/kicad-footprints!3627

Commit:
Added Footprint for TCRT500 from Vishay

See merge request kicad/libraries/kicad-footprints!3499

Commit:
Add: Package_SO: Infineon_PG-DSO-8-59

See merge request kicad/libraries/kicad-footprints!3645

Ignamious committed:
Add the Three different height models of the CK PTS647.

See merge request kicad/libraries/kicad-footprints!3617

johnbeard committed:
Fix Rohm MLGA010V020A pin order

See merge request kicad/libraries/kicad-footprints!3658

Fixes: https://gitlab.com/kicad/libraries/kicad-footprints/-/issues/2646

Commit:
Add/rename Texas DGN0008 variants

See merge request kicad/libraries/kicad-footprints!3646

Commit:
Added footprint for HF115-2Z-x4

See merge request kicad/libraries/kicad-footprints!3634

johnbeard committed:
Generate VSSOP-8_3x3mm_P0.65 (includes rename)

See merge request kicad/libraries/kicad-footprints!3659

johnbeard committed:
Generate VSSOP-8 2.3x2mm P0.5mm, remove 2.4x2.1mm copy

See merge request kicad/libraries/kicad-footprints!3660

pico-pi committed:
Add BC2213A footprint (SSO-8-7, 6.4x9.7mm)

See merge request kicad/libraries/kicad-footprints!3662

johnbeard committed:
Generate SSOP-20_4.4x6.5mm_P0.65mm from IPC script

See merge request kicad/libraries/kicad-footprints!3657

yankee14 committed:
Add TI RVA VFQN-16 from kicad/libraries/kicad-footprint-generator!1139

See merge request kicad/libraries/kicad-footprints!3654

PatrickBaus committed:
Fix: SMA_Samtec_SMA-J-P-H-ST-EM1_EdgeMount

See merge request kicad/libraries/kicad-footprints!3591

johnbeard committed:
Add LQFP-144 EP6.5mm (generated)

See merge request kicad/libraries/kicad-footprints!3648

johnbeard committed:
Update/fix Finder 40 series relays

See merge request kicad/libraries/kicad-footprints!3635

PatrickBaus committed:
Fix ADI DDB/DFN-8-1EP_2x3mm_P0.5mm_EP0.56x2.15mm

See merge request kicad/libraries/kicad-footprints!3650

PatrickBaus committed:
Added 'Linear-DDB' to the keywords of DFN-8-1EP_2x3mm_P0.5mm_EP0.61x2.2mm

See merge request kicad/libraries/kicad-footprints!3665

poeschlr committed:
Add Würth SMTRA horizontal smt mounting hardware

See merge request kicad/libraries/kicad-footprints!1615

miili committed:
Add LFCSP-56-1EP_8x8mm_P0.5mm_EP6.6x6.6mm

See merge request kicad/libraries/kicad-footprints!2305

Commit:
Add Infineon_PG-DSO-8-24 (SOIC-8 variant)

See merge request kicad/libraries/kicad-footprints!2120

Commit:
add Samtec 2.00 mm mPOWER Ultra Micro Power Sockets & Terminals

See merge request kicad/libraries/kicad-footprints!2994

Commit:
NoLead: Re-generate TI RUK package

See merge request kicad/libraries/kicad-footprints!3667

Commit:
Add: generated Macronix_WLCSP-12 (fpgen already merged)

See merge request kicad/libraries/kicad-footprints!3666

Commit:
Add: Molex 205338 2mm pitch Pico-Lock series (fpgen already merged)

See merge request kicad/libraries/kicad-footprints!3668

Commit:
Regenerate VQFN-64-1EP_9x9mm_P0.5mm_EP5.4x5.4mm

See merge request kicad/libraries/kicad-footprints!3670

Commit:
Add: Missing Molex 504050 series 1.5mm Pico-Lock variants (fpgen already merged)

See merge request kicad/libraries/kicad-footprints!3669

Commit:
Add: QFN: Diodes Inc. ZL32 TQFN-32 (fpgen already merged)

See merge request kicad/libraries/kicad-footprints!3673

Commit:
Add: WDFN-8-1EP_8x6mm_P1.27mm_EP6x4.8mm (fpgen already merged)

See merge request kicad/libraries/kicad-footprints!3674

theRealeXa committed:
Added USB_C_Receptable_G-Switch_GT-USB-7025

See merge request kicad/libraries/kicad-footprints!3655

Commit:
Add: BGA: Texas YFP DSBGA-20 package (fpgen already merged)

See merge request kicad/libraries/kicad-footprints!3677

Commit:
Add: Dialog_WLCSP-34_4.54x1.66mm_Stagger (generated)

See merge request kicad/libraries/kicad-footprints!3675

PatrickBaus committed:
Partially added Coilcraft MSS series

See merge request kicad/libraries/kicad-footprints!3679

PatrickBaus committed:
Fixed a typo in the pad size of the Coilcraft MSS1210

See merge request kicad/libraries/kicad-footprints!3681

myfreescalewebpage committed:
LED SMD: Add Inolux IN-P55TATRGB

See merge request kicad/libraries/kicad-footprints!3584

myfreescalewebpage committed:
connector: wuerth: add wr-phd series

See merge request kicad/libraries/kicad-footprints!3683

MacroYau committed:
Add generated MicroCrystal C7 footprint

See merge request kicad/libraries/kicad-footprints!3680

Commit:
Add JST-ZH-ZR connectors (fpgen already merged)

Generator: https://gitlab.com/kicad/libraries/kicad-footprint-generator/-/merge_requests/516

See merge request kicad/libraries/kicad-footprints!3685

hasenradball committed:
Update keywords for SW_Slide-03_Wuerth-WS-SLTV_10x2.5x6.4_P2.54mm (450301014042)

See merge request kicad/libraries/kicad-footprints!3579

Int-Circuit committed:
Adding LEM_LA25-NP

See merge request kicad/libraries/kicad-footprints!3699

Int-Circuit committed:
Fixed LEM_LA25-NP hole and annular ring sizes

See merge request kicad/libraries/kicad-footprints!3711

TiZed committed:
Add SD Card device layout (for 16mm slot depth)

See merge request kicad/libraries/kicad-footprints!3193

Int-Circuit committed:
Adding Finder 40.61 and Finder 40.62 + fixing errors on finder 40.52

See merge request kicad/libraries/kicad-footprints!3698

Kev-J committed:
Add Texas RPU0010A VQFN-HR

See merge request kicad/libraries/kicad-footprints!3708

dos1 committed:
SSOP-10-1EP_3.9x4.9mm_P1mm_EP2.1x3.3mm: Fix 3D model path up

See merge request kicad/libraries/kicad-footprints!3717

jneiva08 committed:
Add HRO USB-C M17

See merge request kicad/libraries/kicad-footprints!3678

johnbeard committed:
Relay_THT: Fix Hongfa HF115F 3D model visibility

See merge request kicad/libraries/kicad-footprints!3725

hasecilu committed:
Adding 3 microswitch footprints from XKB

SW_XKB_DM1-16UC-1
SW_XKB_DM1-16UD-1
SW_XKB_DM1-16UP-1

See merge request kicad/libraries/kicad-footprints!2897

phswoboda committed:
Fix EPC_BGA-4 package pinout

See merge request kicad/libraries/kicad-footprints!3727

jneiva08 committed:
Add QFN-48 7x7mm P0.5mm EP5.7x5.7mm

See merge request kicad/libraries/kicad-footprints!3718

cooked committed:
Add SSOP-4_4.4x2.6mm_P1.27mm from vishay

See merge request kicad/libraries/kicad-footprints!3726

martonmiklos committed:
Add Infineon EasyPACK and ECONO3 footprints

See merge request kicad/libraries/kicad-footprints!3322

JacobOlitsky committed:
Convert pads from rectangle to rounded rectangle

JacobOlitsky committed:
Add flanged version of Amphenol 10118194

Commit:
Add SMA connector BWSMA-KWE-Z001

See merge request kicad/libraries/kicad-footprints!3738

kliment committed:
Fix pad spacing in Coilcraft XAL7030 inductors

See merge request kicad/libraries/kicad-footprints!3742

cpresser committed:
Add smolhaj in scale 1:10

See merge request kicad/libraries/kicad-footprints!3737

cooked committed:
add Texas PicoStar BGA

See merge request kicad/libraries/kicad-footprints!3724

ArcaneNibble committed:
Add QFN-28-1EP_4x4mm_P0.45mm_EP2.6x2.6mm footprint

See merge request kicad/libraries/kicad-footprints!3713

Commit:
Add "DQN0004A" to Texas X2SON 1x1 package metadata

See merge request kicad/libraries/kicad-footprints!3766

Commit:
Added Traco TMF05 footprint

See merge request kicad/libraries/kicad-footprints!3762

cooked committed:
add DFN-6 from script

See merge request kicad/libraries/kicad-footprints!3750

Commit:
Add generated RHB0032M QFN-32

See merge request kicad/libraries/kicad-footprints!3763

Commit:
Added Traco TMF 10 footprint

See merge request kicad/libraries/kicad-footprints!3764

Commit:
Add generated Koherelec MDA5030

See merge request kicad/libraries/kicad-footprints!3760

Commit:
Added Traco TMF20 footprint

See merge request kicad/libraries/kicad-footprints!3765

Bond-009 committed:
Add TerminalBlock CUI TB007-508

See merge request kicad/libraries/kicad-footprints!3756

Commit:
Add generated Texas RNP0030B package

See merge request kicad/libraries/kicad-footprints!3759

Commit:
Add Inductor series: APV ANR

See merge request kicad/libraries/kicad-footprints!3772

Kev-J committed:
Added VSON-14-1EP_3x4.45mm_P0.65mm_EP1.6x4.2mm generated footprint

See merge request kicad/libraries/kicad-footprints!3758

cooked committed:
Add WFDFPN-8

See merge request kicad/libraries/kicad-footprints!3749

Commit:
Analog Devices BGA-165 (11.9x16.0mm, 11x15)

See merge request kicad/libraries/kicad-footprints!3751

Commit:
Add Inductor series: APV APH

See merge request kicad/libraries/kicad-footprints!3773

Commit:
Added Traco TEC3UI footprint

See merge request kicad/libraries/kicad-footprints!3745

Commit:
generated footprints for LFCSP-20-1EP_4x4mm_P0.5mm_EP2.1x2.1mm

See merge request kicad/libraries/kicad-footprints!3753

cooked committed:
add QFN-16 by Melexis

See merge request kicad/libraries/kicad-footprints!3730

jneiva08 committed:
AMASS connector update datasheets link

See merge request kicad/libraries/kicad-footprints!3781

HalfSweet committed:
add: ArtInChip series footprint

See merge request kicad/libraries/kicad-footprints!3747

Commit:
Add Opsco SK6812 5050 addressable LED

See merge request kicad/libraries/kicad-footprints!3779

Commit:
Rework WS2812B with new corner indicator style

See merge request kicad/libraries/kicad-footprints!3783

johnbeard committed:
Regenerate all Gullwing packages with KiCad ordering

See merge request kicad/libraries/kicad-footprints!3686

t-sasatani committed:
Fix u.fl connector (734120110_sd) polarity.

See merge request kicad/libraries/kicad-footprints!3780

phodina committed:
Add Google Coral TPU module

See merge request kicad/libraries/kicad-footprints!3404

johnbeard committed:
Regenerate Nolead (QFN etc) packages with KiCad file ordering

MatthiasKunnen committed:
Fix silkscreen clipped by board edge on USB_Micro-AB_Molex_47590-0001

See merge request kicad/libraries/kicad-footprints!3715

Kev-J committed:
Added Texas REF0038A WQFN-38 NoLead package

See merge request kicad/libraries/kicad-footprints!3777

Commit:
Replace SO-4_7.6x3.6mm_P2.54mm with generated fp

See merge request kicad/libraries/kicad-footprints!3784

Commit:
Add generated RMQ0024A

See merge request kicad/libraries/kicad-footprints!3774 and kicad/libraries/kicad-footprint-generator!1236

Commit:
Add Lemo EPG.00.302.NLN and EPL.00.250.NTN connectors

See merge request kicad/libraries/kicad-footprints!2644

Commit:
Added generated VQFN-100-1EP_12x12mm_P0.4mm_EP8x8mm

See merge request kicad/libraries/kicad-footprints!3791

Kev-J committed:
Added Pulse 4340 series SMD inductors (generated)

See merge request kicad/libraries/kicad-footprints!3790

mampir committed:
Added NSIM-2-C

See merge request kicad/libraries/kicad-footprints!3506

jneiva08 committed:
Add DFN-8-1EP_2x2mm_P0.5mm_EP0.9x1.7mm

See merge request kicad/libraries/kicad-footprints!3707

ferranlala committed:
Add CUI SJ1-351XN

See merge request kicad/libraries/kicad-footprints!2772

Commit:
Update & regenerate QFN-48-1EP_7x7mm_P0.5mm_EP5.6x5.6mm

See merge request kicad/libraries/kicad-footprints!3785

Kev-J committed:
Add Pulse 4332 series inductor

See merge request kicad/libraries/kicad-footprints!3701

Commit:
Added Traco TEN40WIRH footprint

See merge request kicad/libraries/kicad-footprints!3789

mampir committed:
Removed SW_Push_1P1T_NO_6x6mm_H9.5mm

See merge request kicad/libraries/kicad-footprints!3788

Commit:
Added Traco TEN20WIRH footprint

See merge request kicad/libraries/kicad-footprints!3761

ArcaneNibble committed:
Add MYOUNG BS-07-A1BJ001 CR2032 battery holder

See merge request kicad/libraries/kicad-footprints!3767

Kev-J committed:
Added Amphenol 124019772112A USB Type-C waterproof receptacle

See merge request kicad/libraries/kicad-footprints!3776

Commit:
Add Vishay SOP-4-7.5x4.1

See merge request kicad/libraries/kicad-footprints!3793

ryback08 committed:
Add NAIS / Panasonic DR relay series

See merge request kicad/libraries/kicad-footprints!2731

ArcaneNibble committed:
Add QFN-24-1EP_4x4mm_P0.5mm_EP2.5x2.5mm

See merge request kicad/libraries/kicad-footprints!3770

Commit:
Generate & rename DFN-8-1EP_2x2mm_P0.45mm_EP0.64x1.37mm.kicad_mod

See merge request kicad/libraries/kicad-footprints!3629

Nulifier committed:
Added QFN-60 and QFN-80 for RP2350

See merge request kicad/libraries/kicad-footprints!3754

Commit:
Added MaiXu terminal blocks

See merge request kicad/libraries/kicad-footprints!3606

Kev-J committed:
Added DFN-8-1EP_2x2mm_P0.5mm_EP0.86x1.55mm generated by ipc_noLead_generator.py

See merge request kicad/libraries/kicad-footprints!3752

ArcaneNibble committed:
Generate all changjiang FXL inductor footprints

See merge request kicad/libraries/kicad-footprints!3768

Commit:
Add SC-88 term to SOT-363 footprint

See merge request kicad/libraries/kicad-footprints!3797

gcormier committed:
Add FP for TAS5805M

See merge request kicad/libraries/kicad-footprints!3692

Bnjamn committed:
Add Analog devices QFN-28-36 footprint

See merge request kicad/libraries/kicad-footprints!3705

Shackmeister committed:
Add generated TSSOP-28-1EP_4.4x9.7mm_P0.65mm_EP2.74x4.75mm

See merge request kicad/libraries/kicad-footprints!3799

Kev-J committed:
Add XKB U231-091N-4BLRA00-S USB A 3.0 connector

See merge request kicad/libraries/kicad-footprints!2811

Shackmeister committed:
Add Generated Texas_PWP0028V_TSSOP-28-1EP_4.4x9.7mm_P0.65mm_EP3.4x9.7mm_Mask2.94x5.62mm + Thermalvia

See merge request kicad/libraries/kicad-footprints!3800

Shackmeister committed:
Add Generated TSSOP-28-1EP_4.4x9.7mm_P0.65mm_EP3.05x7.56mm + Thermalvias

See merge request kicad/libraries/kicad-footprints!3802

Commit:
Add Texas RSW package (uQFN-10)

See merge request kicad/libraries/kicad-footprints!3640

sergiobenrocha2 committed:
Added SOP-44_13.3x28.2mm_P1.27mm (generated)

See merge request kicad/libraries/kicad-footprints!3803

Commit:
Fixes #2648 - Update Datasheet URL for LFCSP-16-1EP_4x4mm_P0.65mm_EP2.6x2.6mm

See merge request kicad/libraries/kicad-footprints!3805

cooked committed:
add VSON-8 EP1.6x2.4mm

See merge request kicad/libraries/kicad-footprints!3723

Commit:
Adds WSON-10-1EP_4x4mm_P0.8mm_EP2.6x3mm (generated)

See merge request kicad/libraries/kicad-footprints!3804

Commit:
Replace WSON-10-1EP_2x3mm_P0.5mm_EP0.84x2.4mm by generated package

See merge request kicad/libraries/kicad-footprints!3806

ArcaneNibble committed:
Add QFN-28-1EP_4x4mm_P0.4mm_EP2.8x2.8mm footprint (generated)

See merge request kicad/libraries/kicad-footprints!3714

phodina committed:
Add QFN-64-1EP_9x9mm_P0.5mm_EP5.7x5.7mm generated using the footprint-generator

See merge request kicad/libraries/kicad-footprints!3719

hasenradball committed:
Updated Wemos Mini D1 Module

See merge request kicad/libraries/kicad-footprints!3580

Shackmeister committed:
Delete: TSSOP-28-1EP_4.4x9.7mm_P0.65mm (replaced by more specific generated footprints)

See merge request kicad/libraries/kicad-footprints!3807

kacper-ka committed:
Add Espressif ESP32-C3-DevKitM-1

See merge request kicad/libraries/kicad-footprints!2953

johnbeard committed:
Regenerate DPAKs (new format and fixed silk markers/clearance)

See merge request kicad/libraries/kicad-footprints!3808

NetSpida committed:
Manual soldering footprint for folded 0.42" OLED Display with SSD1306 driver, 72x40 points.

See merge request kicad/libraries/kicad-footprints!3141

mampir committed:
Renamed WA-SMRA to WP-SMRA, rounded pads

See merge request kicad/libraries/kicad-footprints!3810

Commit:
Added Traco TMF 30 footprint

See merge request kicad/libraries/kicad-footprints!3801

mampir committed:
Split Relay_DPDT_Kemet_EC2 into NU and NJ variants

See merge request kicad/libraries/kicad-footprints!3809

johnbeard committed:
Add: Wuerth 7999010211A RJ45 magjack

See merge request kicad/libraries/kicad-footprints!3524

johnbeard committed:
Regenerate BGA packages with new generator formatting

Commit:
Add CFSensor XGZP6859D

See merge request kicad/libraries/kicad-footprints!3605

johnbeard committed:
Revert regeneration of manually-adjusted part: UQFN-16_1.8x2.6mm_P0.4mm.kicad_mod

johnbeard committed:
UQFN-16: Regenerate with proper pad override

See merge request kicad/libraries/kicad-footprints!3814

cpresser committed:
Fix silkscreen on 3 non-generated BGA footprints

Closes: #2639

phswoboda committed:
Add ST-PowerFLAT_8x8

See merge request kicad/libraries/kicad-footprints!3702

MatthiasKunnen committed:
Fix Potentiometer_Alps_RK097_Single_Horizontal silkscreen

See merge request kicad/libraries/kicad-footprints!3716

johnbeard committed:
Add generated QFN 1.8x2.6mm P0.4mm (STG3692)

See merge request kicad/libraries/kicad-footprints!3484

Commit:
Add JST XH side entry SMD connectors

See merge request kicad/libraries/kicad-footprints!3636

jneiva08 committed:
Add Microchip PQFP-44

See merge request kicad/libraries/kicad-footprints!3827

tobiasfalk committed:
Add LFCSP-24-1EP_4x4mm_P0.5mm_EP0.5x0.5mm (generated)

See merge request kicad/libraries/kicad-footprints!3831

hasenradball committed:
add ESP-01 wifi module

See merge request kicad/libraries/kicad-footprints!3483

sergiobenrocha2 committed:
Add PZB300 EMI suppressor

See merge request kicad/libraries/kicad-footprints!3832

mojca committed:
Set .kicad_mod and some others to text in .gitattributes

Having the type set to 'text' rather than 'text=auto' enforces git
to always normalize line endings on those files.

mojca committed:
Remove obsolete files

See merge request kicad/libraries/kicad-footprints!3836

mojca committed:
Regenerate Inductor Pulse SMT PA43xx after fixing a typo

See merge request kicad/libraries/kicad-footprints!3840

johnbeard committed:
TO_SOT_THT: Regenerate with v8 generator scripts

See merge request kicad/libraries/kicad-footprints!3817

Kev-J committed:
Added Pulse PA5402 SMD inductor

See merge request kicad/libraries/kicad-footprints!3841

johnbeard committed:
DIP: regenerate with v8 generator format updates

See merge request kicad/libraries/kicad-footprints!3815

mojca committed:
Fix datasheet URLs for some coaxial connectors

See merge request kicad/libraries/kicad-footprints!3845

mojca committed:
Apply automatic changes made by KiCad

See merge request kicad/libraries/kicad-footprints!3845

Kev-J committed:
Added Pulse PA4341 generated footprint

See merge request kicad/libraries/kicad-footprints!3816

Shackmeister committed:
Replace handmade ST_UQFN-6L_1.5x1.7mm_P0.5mm with generated footprint

See merge request kicad/libraries/kicad-footprints!3830

Kev-J committed:
Added Hirose FX8 series connectors (generated)

See merge request kicad/libraries/kicad-footprints!3748

sergiobenrocha2 committed:
Add SOP-32, dimensions from R1LP0108E (generated)

See merge request kicad/libraries/kicad-footprints!3846

ArcaneNibble committed:
Add Anpec_WLCSP-20_1.76x2.03mm_P0.4mm (generated)

See merge request kicad/libraries/kicad-footprints!3769

Commit:
Added Coilcraft Air core SMD inductor series SQ 1515, 2222, 2929 (generated)

See merge request kicad/libraries/kicad-footprints!3844

Commit:
Added Traco THR 40WI footprint (generated)

See merge request kicad/libraries/kicad-footprints!3839

tobiasfalk committed:
Add DIP-24_18.0mmx34.29mm_W15.24mm

See merge request kicad/libraries/kicad-footprints!3847

phswoboda committed:
Add ST PowerFLAT 5x6 HV footprint

See merge request kicad/libraries/kicad-footprints!3704

Commit:
Re-generate PLCCs using PLCC generator

See merge request kicad/libraries/kicad-footprints!3643

Commit:
Generate previously manually drawn UQFN-10_1.4x1.8mm_P0.4mm

See merge request kicad/libraries/kicad-footprints!3644

JacobOlitsky committed:
Fix TE Mate-N-Lok horizontal connector pin orders

See merge request kicad/libraries/kicad-footprints!3744

Fixes #2650

yankee14 committed:
Fix ESP32-S2_WROVER EP paste pad to be segmented

See merge request kicad/libraries/kicad-footprints!3656

mojca committed:
Update footprints after changes in generator

See merge request kicad/libraries/kicad-footprint-generator!1274

Ignamious committed:
Add generated footprint for TQFP-64-1EP_10x10mm_P0.5mm_EP5.305x5.305mm.

See merge request kicad/libraries/kicad-footprints!3652

mojca committed:
Regenerate the footprints for SOT-23-*

See merge request kicad/libraries/kicad-footprint-generator!1297
See merge request kicad/libraries/kicad-footprint-generator!1299
See merge request kicad/libraries/kicad-footprints!3854

mojca committed:
Regenerate footprint with fixed pad sorting

See merge request kicad/libraries/kicad-footprints!3850

mojca committed:
Regenerate footprints with fixed pad sorting

See merge request kicad/libraries/kicad-footprints!3851

mojca committed:
Regenerate the footprint for SOT-23

See merge request kicad/libraries/kicad-footprints!3853

mojca committed:
Update the datasheet link for BGA-25

Also apply automatic changes added by KiCad.

See merge request kicad/libraries/kicad-footprints!3856

mojca committed:
Delete BGA-121_12.0x12.0mm_Layout11x11_P1.0mm

The datasheet at the new URL
  https://www.analog.com/media/en/package-pcb-resources/package/pkg_pdf/ltc-legacy-bga/05081891_B_bga121.pdf
seems to suggest that the part has pitch 1.27mm rather than 1.0mm,
and the package is 15.0x15.0mm rather than 12.0x12.0mm wide.

See kicad/libraries/kicad-packages3D#542
See merge request kicad/libraries/kicad-packages3D!1085
See merge request kicad/libraries/kicad-footprints!3856

cpresser committed:
Rework LED_SMD Pin1 silkscreen markers

See merge request kicad/libraries/kicad-footprints!3819

Commit:
Remove B.Paste layer from pad 5 of ST_PowerFLAT-6L_5x6mm_P1.27mm

Closes kicad/libraries/kicad-footprints#2664
See merge request kicad/libraries/kicad-footprints!3843

sergiobenrocha2 committed:
Fix typo in SOP-32 name (11.305mm width), it belongs to sop not ssop

See merge request kicad/libraries/kicad-footprints!3855

sergiobenrocha2 committed:
Added 300 / 400 mils SOJ footprints (used in SRAMs), fixed SOJ-36_10.16x23.49mm_P1.27mm

See merge request kicad/libraries/kicad-footprints!3848

mojca committed:
Regenerate Samtec connectors

See merge request kicad/libraries/kicad-footprint-generator!1306
See merge request kicad/libraries/kicad-footprints!3859

gkeeth committed:
THT LEDs fixup (regen after generator changes)

See merge request kicad/libraries/kicad-footprints!3842

sergiobenrocha2 committed:
Added Footprint Alliance Memory TFBGA-36 for AS6C1008 SRAM (generated)

See merge request kicad/libraries/kicad-footprints!3858

johnbeard committed:
Correct pad numbers in Harwin Gecko footprints

While the datasheet only specifies pin 1, a supplementary
document also shows a specific numbering order (down one
row and the down the other, in the same direction).

Fixes: https://gitlab.com/kicad/libraries/kicad-footprints/-/issues/2655

johnbeard committed:
Gecko connectors: update silk/fab, fix courtyards, URLs

Kev-J committed:
Added Kycon KUSBX-AS1N-B USB type A connector

See merge request kicad/libraries/kicad-footprints!3690

Int-Circuit committed:
Add Sipeed-M1W and fix Sipeed-M1 fp

See merge request kicad/libraries/kicad-footprints!3691

mojca committed:
Delete LED_Rectangular_W3.9mm_H1.8mm_FlatTop

The footprint seems to be the same as for
LED_Rectangular_W3.9mm_H1.8mm, and that one is already "flat enough".

The datasheet points to L-2774GD which officially has a round top.
Maybe perfectly rectangular ones with a flat top are being sold,
but it feels strange to have a duplicate footprint without even a 3D model
and relatively unusual name.

mojca committed:
Add BGA-196_15x15mm_Layout14x14_P1.0mm_Ball0.5mm (generated)

See merge request kicad/libraries/kicad-footprints!3860

jjm2473 committed:
Add TFBGA-636 for RK3288/RTD1296 (generated).

See merge request kicad/libraries/kicad-footprints!2472

phodina committed:
Add VQFN-28-1EP_5x5mm_P0.5mm_EP3.25x3.25mm (generated)

See merge request kicad/libraries/kicad-footprints!3866

cnieves1 committed:
Add Bourns SRP6060FA, update SRP1050WA and SRP5030T

See merge request kicad/libraries/kicad-footprints!3864

johnbeard committed:
Regenerate QFNs with latest format code

See merge request kicad/libraries/kicad-footprints!3868

phodina committed:
Add Quectel BG95

See merge request kicad/libraries/kicad-footprints!3867

johnbeard committed:
QFN: generate UQFN-20_3x3mm_P0.4mm

The generator for this was merged by
https://gitlab.com/kicad/libraries/kicad-footprint-generator/-/merge_requests/518

kliment committed:
Regenerate and rename OnSemi_UDFN-14-1EP_4x4mm_P0.5mm_EP2.7x3.4mm to OnSemi_DFN-14-1EP_4x4mm_P0.5mm_EP2.7x3.4mm

See merge request kicad/libraries/kicad-footprints!3870

johnbeard committed:
Add: DFN-4 5x7mm with keepout (DSC1001, generated)

See merge request kicad/libraries/kicad-footprints!3647

myfreescalewebpage committed:
Add UQFN-20-1EP_3x3mm_P0.4mm_EP1.7x1.7mm

See merge request kicad/libraries/kicad-footprints!3582

Commit:
add ST_WLCSP-41_2.98x2.76mm_P0.40mm_Stagger (generated)

See merge request kicad/libraries/kicad-footprints!3871

Igorbunow committed:
Add Diodes_UDFN3810-9_TYPE_B footprint

See merge request kicad/libraries/kicad-footprints!2657

aris-kimi committed:
Correct Diodes_UDFN3810-9_TYPE_B 3D path

See merge request kicad/libraries/kicad-footprints!3872

mojca committed:
Add a few more text filetypes to .gitattributes

See merge request kicad/libraries/kicad-footprints!3873

Commit:
Added Traco TSR 0.6WI footprint

See merge request kicad/libraries/kicad-footprints!3838

sergiobenrocha2 committed:
Added SOIC-32 (300 mils width), JEDEC MO-119-B, variation AC

See merge request kicad/libraries/kicad-footprints!3879

Commit:
add Texas_PicoStar_DFN-3_0.69x0.60mm

See merge request kicad/libraries/kicad-footprints!3878

Shackmeister committed:
Molex 43650: regenerated with mounting pads attributes

See merge request kicad/libraries/kicad-footprints!3798

ArcaneNibble committed:
Add Everlight IRM-H6xxT infrared remote receiver

See merge request kicad/libraries/kicad-footprints!3712

ademuri committed:
Set the zone connection override to "solid" for ESP32 thermal vias. Also set the heatsink fabrication property for these vias where not already set.

See merge request kicad/libraries/kicad-footprints!3706

Kev-J committed:
Add JAE WP7B series FPC connector footprints generated by connector_generator.py

See merge request kicad/libraries/kicad-footprints!3746

sergiobenrocha2 committed:
Added TFBGA-48 and infineon variant (generated)

See merge request kicad/libraries/kicad-footprints!3880

Commit:
add Crystal_SMD_1210-4Pin_1.2x1.0mm footprint

See merge request kicad/libraries/kicad-footprints!3875

Commit:
Add Texas_DSBGA-6_0.757x1.01mm_Layout2x3_P0.35mm (generated)

See merge request kicad/libraries/kicad-footprints!3882

Commit:
Update Texas_S-PDSO-N12 footprint to match TI recommendation

See merge request kicad/libraries/kicad-footprints!3874

aris-kimi committed:
Add C_1808_4520Metric (generated)

See merge request kicad/libraries/kicad-footprints!3820

phswoboda committed:
Add EPC23102 eGaN Power-Stage

See merge request kicad/libraries/kicad-footprints!3532

Commit:
renamed ST_WLCSP-42_2.93mmx2.82_P0.40mm_Stagger to ST_WLCSP-42_2.93x2.82mm_P0.40mm_Stagger

See merge request kicad/libraries/kicad-footprints!3885

jneiva08 committed:
Add ORG1510

See merge request kicad/libraries/kicad-footprints!3096

jneiva08 committed:
Add USB-C Amphenol 12401948E412A

See merge request kicad/libraries/kicad-footprints!3540

knst-wrk committed:
Connector_SFP:Change pin 1 marking on fab from chamfer to triangle due to ambiguity with some vendor parts

See merge request kicad/libraries/kicad-footprints!3889

sergiobenrocha2 committed:
Added QFN-28-1EP_5x5mm[...] footprints from JEDEC and Silicon Labs

See merge request kicad/libraries/kicad-footprints!3890

Commit:
Added Traco TEN 10WIRH footprint

See merge request kicad/libraries/kicad-footprints!3888

sergiobenrocha2 committed:
Added Infineon_SOIC-20W_7.6x12.8mm_P1.27mm new footprint, compatible with Infineon PG-DSO-20

See merge request kicad/libraries/kicad-footprints!3893

fmunier committed:
Add Texas_PW0020A_HTSSOP-20_4.4x6.5mm_P0.65mm (generated)

See merge request kicad/libraries/kicad-footprints!3887

fmunier committed:
Rename Texas_PW0020A_HTSSOP-20_4.4x6.5mm_P0.65mm.kicad_mod -> Texas_PW0020A_TSSOP-20_4.4x6.5mm_P0.65mm.kicad_mod

See merge request kicad/libraries/kicad-footprints!3896

See also: https://gitlab.com/kicad/libraries/kicad-footprint-generator/-/merge_requests/1345

chmorgan committed:
SiTime_PQFN-4L oscillator footprint

See merge request kicad/libraries/kicad-footprints!3891

sergiobenrocha2 committed:
Added Mini Relay K (Sealed Version) / V23072 from Tyco Electronics

See merge request kicad/libraries/kicad-footprints!3900

Shackmeister committed:
Add "Heatsink Pad" Attribute to all SMD USB Mountingpads to avoid DRC error

See merge request kicad/libraries/kicad-footprints!3899

johnbeard committed:
SSOP: Update DS for SSOP-8_3.9x5.05mm_P1.27mm

See merge request kicad/libraries/kicad-footprints!3894

johnbeard committed:
Add Texas DAD0032A and DKD0036A TopEP footprints (generated)

See merge request kicad/libraries/kicad-footprints!3895

Shackmeister committed:
Pin 1 tweaks: OnSemi_Micro8 & Vishay_PowerPAK_1212-8_Single&Dual

See merge request kicad/libraries/kicad-footprints!3898

Commit:
Add generated JUSHUO AFA07 FFC connectors

See merge request kicad/libraries/kicad-footprints!3559

zitronenoeffner committed:
add generated footprint QFN-28-1EP_4x4mm_P0.4mm_EP2.7x2.7mm and remove obsolete fp QFN-28-1EP_4x4mm_P0.4mm_EP2.8x2.8mm

See merge request kicad/libraries/kicad-footprints!3901

johnbeard committed:
Inductor_SMD: Regenerate with correct pad rounding

See merge request kicad/libraries/kicad-footprints!3903

XDjackieXD committed:
Add ESP32-C6-Mini-1

See merge request kicad/libraries/kicad-footprints!3884

Commit:
Add generated WCH-QFN16

See merge request kicad/libraries/kicad-footprints!3902

Commit:
Add generated bridge rectifier MB8S

See merge request kicad/libraries/kicad-footprints!3795

gmarull committed:
Add RAK3172 footprint

See merge request kicad/libraries/kicad-footprints!3886

johnbeard committed:
SOT-553/563: Generate with flat-lead-aware gullwing generator

See merge request kicad/libraries/kicad-footprints!3906

MacroYau committed:
Add Nordic nRF9160 LGA footprint

See merge request kicad/libraries/kicad-footprints!3905

cnieves1 committed:
Added Jadak Thingmagic M6e-Nano footprint

See merge request kicad/libraries/kicad-footprints!3909

cnieves1 committed:
Added Telit SE150A4 smart module

See merge request kicad/libraries/kicad-footprints!3194

Commit:
Traco TEN 10WIRH added compatible mpns

See merge request kicad/libraries/kicad-footprints!3912

knst-wrk committed:
Add a few missing WAGO 236 series terminal lengths, regenerate remaining WAGO 236 series.

See merge request kicad/libraries/kicad-footprints!3913
